cmd/handoff-bss-metadata.go

```diff
@@ -481,29 +514,80 @@ func getPITVLanString(vlan string) string {
 	return string(out)
 }
 
-func populateNCNMetadata() {
-	var err error
-	var ncnRootPassword string
-
-	bssEntries := make(map[string]bssTypes.BootParams)
+func publicKeysCallback() (signers []ssh.Signer, err error) {
+	// Use public key auth for SSH if we can set it up...
+	key, err := os.ReadFile(os.Getenv("HOME") + "/.ssh/id_rsa")
```

Comment:
I'm wondering if we should have rsa be the default, but allow another type via env var override?

Reply:
I could probably add logic to go through all of the private keys found in .ssh like SSH actually does. Have to find out how Go does that, but I suspect it is just an OS library call of some kind. That would be better than driving it from an env var, I think.

Reply:
I have tested the new approach and it works. I put in a few different keys and it found the one that worked and complained about then skipped the one that was bogus.